modules/openshift-developer-cli-deploying-the-back-end-component.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-multicomponent-application-with-odo.adoc
+
+[id="deploying-the-back-end-component_{context}"]
+
+= Deploying the back-end component
+
+.Procedure
+
+. Import `openjdk18` into the cluster:
++
+----
+$ oc import-image openjdk18 \
+--from=registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift --confirm
+oc annotate istag/openjdk18:latest tags=builder
+----
+
+. Tag the image as `builder` to make it accesible for {odo-title}:
++
+----
+$ oc annotate istag/openjdk18:latest tags=builder
+----
+
+. Run `odo catalog list components` to see the created image:
++
+----
+$ odo catalog list components
+Odo Supported OpenShift Components:
+NAME          PROJECT       TAGS
+nodejs        openshift     10,8,8-RHOAR,latest
+openjdk18     myproject     latest
+----
+
+. Change directory to the back-end source directory and check that you have the correct files in the directory:
++
+----
+$ cd ~/backend
+$ ls
+debug.sh  pom.xml  src
+----
+
+. Build the back-end source files with Maven to create a JAR file:
++
+----
+$ mvn package
+...
+[INFO] --------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] --------------------------------------
+[INFO] Total time: 2.635 s
+[INFO] Finished at: 2019-09-30T16:11:11-04:00
+[INFO] Final Memory: 30M/91M
+[INFO] --------------------------------------
+----
++
+Now {odo-title} can run atop the Java application server.
```

Comment:
I don not understand the sense of the sentence. Can you please look into this. cc_ @cdrage

Reply:
Me neither.. I don't remember if I wrote this, but could you remove it @boczkowska ?

Reply:
removed